lingvo/jax/BUILD

```diff
@@ -1,7 +1,7 @@
 # Description:
 #   Base directory for the lingvo Jax library.
 
-load("//lingvo:lingvo.bzl", "pytype_strict_binary", "pytype_strict_library")
+load("//lingvo:lingvo.bzl", "pytype_strict_binary", "pytype_strict_library", "lingvo_proto_py")
```

Comment:
Please make this a separate line/instruction because of internal rule rewrite:
load("//lingvo:lingvo.bzl", "lingvo_proto_py")

Reply:
Thanks. Done.